src/main/java/net/dv8tion/jda/api/entities/Member.java

```diff
@@ -129,6 +130,21 @@
     @Nonnull
     OnlineStatus getOnlineStatus(@Nonnull ClientType type);
 
+    /**
+     * A Set of all active {@link net.dv8tion.jda.api.entities.ClientType ClientTypes} of this Member.
+     * Every {@link net.dv8tion.jda.api.OnlineStatus OnlineStatus} other than {@code OFFLINE} and {@code UNKNOWN}
+     * is interpreted as active.
+     * <br>If the Member is currently not active with any Client, this returns an empty Set.
```

Comment:
Mention `INVISIBLE` here as well since some people might think its tracked because its present in the enum

Reply:
Also it should be pointed out that this will always be empty if the `CacheFlag.CLIENT_STATUS` is disabled.